src/data_hub/plotting.py

```diff
@@ -109,8 +146,9 @@ def scatter(
         y="y",
         source=scatter_source,
         size="size",
-        color="blue",
+        color="colors",
```

Comment:
this is highly confusing, but this now refers to a column called `colors` in the CDS, not the literal string `blue` as it previously did